provider/blob/src/export/mod.rs

```diff
@@ -22,7 +22,11 @@
 //! // Export something
 //! DatagenDriver::new()
 //!     .with_keys([icu_provider::hello_world::HelloWorldV1Marker::KEY])
-//!     .with_all_locales()
+//!     .with_locales_and_fallback([LocaleFamily::FULL], {
+//!         let mut options = FallbackOptions::default();
+//!         options.deduplication_strategy = Some(DeduplicationStrategy::None);
+//!         options
+//!     })
```

Comment:
Suggestion (optional): since `BlobExporter` does not support runtime fallback, `DeduplicationStrategy::None` is the default, so you don't need to set it explicitly here